src/block/inventory/BrewingStandInventory.php

```diff
@@ -39,4 +40,20 @@ public function __construct(Position $holder, int $size = 5){
 		$this->holder = $holder;
 		parent::__construct($size);
 	}
+
+	public function getIngredient() : Item{
+		return $this->getItem(self::SLOT_INGREDIENT);
+	}
+
+	public function setIngredient(Item $item) : void{
+		$this->setItem(self::SLOT_INGREDIENT, $item);
+	}
+
+	public function getFuel() : Item{
+		return $this->getItem(self::SLOT_FUEL);
+	}
+
+	public function setFuel(Item $item) : void{
+		$this->setItem(self::SLOT_FUEL, $item);
+	}
```

Comment:
These methods are redundant. It's sufficient to just use the constants with setItem().